packages/vscode-extension/src/panels/WebviewController.ts

```diff
@@ -51,22 +49,18 @@ export class WebviewController implements Disposable {
     this.setWebviewMessageListener(webview);
 
     // Set the manager to listen and change the persisting storage for the extension.
-    this.dependencyChecker = new DependencyChecker(webview);
-    this.dependencyChecker.setWebviewMessageListener();
-
-    this.dependencyInstaller = new DependencyInstaller(webview);
-    this.dependencyInstaller.setWebviewMessageListener();
+    this.dependencyManager = new DependencyManager(webview);
+    this.dependencyManager.setWebviewMessageListener();
```

Comment:
Why we don't do it in the c-tor?

Reply:
do you think somethnig like `isNpmModulesInstalled` makes more sense?